src/Hidden/Hidden.test.tsx

```diff
@@ -36,7 +37,8 @@ describe('Hidden', () => {
   })
   it('renders the styles as expected when a single viewport value is provided as a string via `on` prop', () => {
     const expectedStyles = {
-      [`@media screen and (min-width:768px)`]: {
+      // `.replace` is used because renderStyles return the JSON object without a space after the column
+      [`@media screen and ${viewportRanges.regular.replace(': ', ':')}`]: {
```

Comment:
`replace` 😢  Any alternatives?

Reply:
@broccolinisoup could we have a `breakpoint` helper like:

```suggestion
      [breakpoint(viewportRanges.regular)]: {
```

Let me know if I'm missing anything! For context, this was the setup/structure I was thinking of from Carbon: https://github.com/carbon-design-system/carbon/blob/main/packages/layout/src/index.js#L43-L83 definitely don't mean we should copy it by any means, just was thinking about how the breakpoints were structured and the helper functions that made it simple to emit media queries in JS

Reply:
Thank you! I still needed `replace` but took your suggestion into account and came up with something similar. Here is the [commit](https://github.com/primer/react/pull/2729/commits/37fd3b854c9d4d4645ff8b7be3deb8880df4b7eb) 

If that looks good, I will replace the `mediaQueriesForViewportRanges` with the hard-coded values. Also, open to name suggestions 🙌🏻